.github/workflows/travis.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
```

Comment:
The file should be renamed to `test.yml`. Travis is the name of a CI that we don't want to rely on anymore.